content/blog/2020-07-10-july-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,150 @@
+
+## Community activity
+
+We've been busy! Although we are mostly homebound these days, there has been no
+shortage of speaking engagements. Here's a recap.
+
+Co-founders Dmitry and Ivan appeared on the HasGeek TV series
+[Making Data Science Work](https://hasgeek.com/fifthelephant/making-data-science-work-session-3/)
+to discuss engineering for data science with hosts
+[Venkata Pingali](https://www.linkedin.com/in/pingali/) and
+[Indrayudh Ghoshal](https://www.linkedin.com/in/indrayudhghoshal/). The
+livestream is available for viewing on YouTube!
+
+https://www.youtube.com/watch?v=EWcpALbzZRg
+
+The same day, Dmitry appeared on the [MLOps.community](https://mlops.community/)
+meetup to chat with host
+[Demetrios Brinkmann](https://www.linkedin.com/in/dpbrinkm/). They talked about
+the open source ecosystem, the difference between tools and platforms, and what
+it means to codify data.
+
+https://www.youtube.com/watch?v=ojV1tK9jXH8&t=2295s
+
+I (Elle) gave another talk at the
+[MLOps Production & Engineering World](https://mlopsworld.com/) meeting, called
+"Adapting continuous integration and continuous delivery for ML". I shared an
+approach to using GitHub Actions & GitLab CI to start using continuous
+integration in ML model training- and it turns out, this conference is generally
+very excited about GitHub Actions! In another talk, GitHub technical advocate
+[Jonathan Peck](https://www.linkedin.com/in/peckjon/), shared some great use
+cases, including how he used an Nvidia Jetson Nano to run Actions. Generally,
+the mood around GitHub Actions in the ML community is very encouraging for a
+future with more automated data science practices.
+
+Extremely early the next morning, I co-led a workshop with clinician-scientist
+[Cris Lanting](https://www.linkedin.com/in/crislanting/?originalSubdomain=nl)
+about developing strong computational infrastructure and practices in research
+as part of the
+[Virtual Conference on Computational Audiology](https://computationalaudiology.com/).
+
+In addition to outlining key components of scientific reproducibility and
+project management, Cris and I did some investigating. We conducted an informal
+survey asking about the biggest barriers to making research reproducible and
+shareable. Participants overwhelmingly answered that they lack time and
+incentives. Difficulty managing dependencies (both data and software) was
+another popular answer. Sounds a bit like softare before
+[DevOps](https://en.wikipedia.org/wiki/DevOps)... For the curious, the workshop
+is still viewable!
+
+https://www.youtube.com/watch?v=W4CoptalWw0
+
+Lastly, check out our virtual poster for
+[SciPy 2020](https://www.scipy2020.scipy.org/)! We prepared a demo about
+[packaging models and datasets like software](https://dvc.org/blog/scipy-2020-dvc-poster)
+so they can be widely disseminated via GitHub.
+
+## Coming up soon
+
+Some things to watch out for in the next few weeks:
+
+- **DVC Virtual Meetup.** In May, we had our
```

Comment:
let's make it at the very begging? it would require some slight change in the structure but it feels this is one of the most important parts for us

Reply:
that'll work!

Reply:
I would even mention in the description/descriptionLong - 📣 Join DVC Meetup!! ...

Reply:
great idea